sticky_notes.py

```diff
@@ -81,3 +81,26 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+
+    @cmd
+    def del_note(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) == 1:
+            nick = args[0]
+
+            if nick in self.notes:
+                notes = self.notes[nick]
+                for i in range(len(notes)):
+                    if from_nick == notes[i].sender:
+                        to_pop = i
+                        break
+                notes.pop(to_pop)
```

Comment:
So if `from_nick == notes[i].sender` is not true for any `i` then `to_pop` won't be defined and this line will just throw an exception?